package.json

```diff
@@ -32,8 +33,9 @@
     "vuedraggable": "^2.23.2"
   },
   "scripts": {
-    "test": "npx eslint .",
+    "test": "npx eslint . --ext .js,.vue",
     "storybook": "start-storybook -s ./public -p 9001 -c .storybook",
+    "dev": "start-storybook -s ./public -p 9001 -c .storybook",
```

Comment:
My terminal constantly asks me `zsh: correct 'storybook' to '.storybook' [nyae]?`. I'd love the easy option to skip that.